src/rpc/mining.cpp

```diff
@@ -441,15 +460,20 @@ UniValue auditsnapshotaccrual(const UniValue& params, bool fHelp)
 
     int64_t period = tally_accrual_period("tip", 0, pindex_low->nTime, now, 0);
 
+    result.pushKV("cpid", cpid->ToString());
```

Comment:
I might rearrange things a bit, but not right now, because yes, they are not laid out optimally, but I am using the output for analysis, and I don't want to change the analytical parts downstream. I will some cleanup in a subsequent PR. I am doing test syncs from zero to make sure this didn't break anything before merging this.